src/test/e2e/services/access-service.e2e.test.ts

```diff
@@ -1382,13 +1383,15 @@ test('calling add access with invalid project role ids should not assign those r
 
     const adminRootRole = await accessService.getRoleByName(RoleName.ADMIN);
 
-    accessService.addAccessToProject(
-        [adminRootRole.id, 9999],
-        [],
-        [emptyUser.id],
-        projectName,
-        'some-admin-user',
-    );
+    expect(async () =>
```

Comment:
await expect(() => ...)

Reply:
done in d8851f8fcd7174d4c08f4aa71804dda39c3bacbe